modules/update-upgrading-web.adoc

```diff
@@ -22,18 +27,39 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to the correct
-`stable-<major_version>` channel for the version that you want to update to.
+. For production clusters, ensure that the *CHANNEL* is set to the correct
+`stable-*` channel for
+ifdef::within[]
+the version that you want to update to.
+endif::within[]
+ifdef::between[]
+your current minor version.
+endif::between[]
 +
 [IMPORTANT]
 ====
 For production clusters, you must subscribe to a `stable-*` channel.
 ====
-.. If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
 cluster.
-.. The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** The *DESIRED VERSION* indicates the cluster version that your cluster is running
 or is updating to.
 
-. Click *Updates Available*, select a version to update to, and click *Update*.
+. Click *Updates Available*, select
+ifdef::within[]
+a version to update to,
+endif::within[]
+ifdef::between[]
+the highest available version
```

Comment:
One hop does not guarantee you get to a good jumping-off point.  For example, the current stable-4.1 graph is:

```console
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=stable-4.1' | ~/src/openshift/cincinnati/hack/graph.sh | dot -Tpng >graph.png
```

using [`graph.sh`][1] to generate:

![graph](https://user-images.githubusercontent.com/209920/66527878-86505280-eab2-11e9-9382-8657d9cfab4f.png)

With 4.1.18 as a jumping-off point (as we're currently planning), folks on 4.2.0 would have to go 4.2.0->4.2.14->4.1.18 or similar.  Anyone still running the guarded 4.1.10 would have to go 4.1.10->4.1.11->4.1.18 or similar.  We probably want to say "and keep upgrading until there are no recommended upgrades left" somewhere for this use-case.

[1]: https://github.com/openshift/cincinnati/blob/master/hack/graph.sh

Reply:
Is there a better way to get that graph and see what updates are available? "and keep upgrading until there are no recommended upgrades left" isn't the best user experience.

Reply:
Not at the moment, but maybe in the mid-term.  Eventual UX will be "turn on auto-upgrades, set your target channel, and we'll route you through the graph".

Reply:
When that's about to happen, please let me know.